include/unifex/linux/io_uring_context.hpp

```diff
@@ -942,6 +956,171 @@ inline io_uring_context::scheduler io_uring_context::get_scheduler() noexcept {
   return scheduler{*this};
 }
 
+class io_uring_context::accept_sender {
+  using offset_t = std::int64_t;
+
+  template <typename Receiver>
+  class operation : private completion_base {
```

Comment:
A little surprising that this is a class template that's used directly, rather than using a nested, non-template `type`.

Reply:
#533